src/Microsoft.ML.Data/Transforms/LabelIndicatorTransform.cs

```diff
@@ -111,6 +116,23 @@ private static string TestIsMulticlassLabel(ColumnType type)
             return $"Label column type is not supported for binary remapping: {type}. Supported types: key, float, double.";
         }
 
+        /// <summary>
+        /// Convenience constructor for public facing API.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="name">Name of the output column.</param>
+        /// <param name="source">Name of the input column.  If this is null '<paramref name="name"/>' will be used.</param>
+        /// <param name="classIndex">Label of the positive class.</param>
+        public LabelIndicatorTransform(IHostEnvironment env,
+            IDataView input,
+            string name,
+            string source = null,
+            int classIndex = Defaults.ClassIndex)
```

Comment:
This ought to have been a required parameter.

Reply:
I see that zero is being used as default value. Do you suggest making it required?